src/util/errorObject.ts

```diff
@@ -1 +1,2 @@
-export var errorObject = { e: {} };
\ No newline at end of file
+// Any so that it we don't have to cast when comparing it's values.
+export var errorObject: any = { e: {} };
```

Comment:
@kwonoj @luisgabriel this is the change I was talking about.

The object is the same, but the type is now just `any`.  I've stripped out any of the remaining `as any` casts as well.